src/llmtuner/hparams/generating_args.py

```diff
@@ -46,7 +46,10 @@ class GeneratingArguments:
         default=1.0,
         metadata={"help": "Exponential penalty to the length that is used with beam-based generation."},
     )
-
+    stop: Union[Optional[str], List[str]] = field(
```

Comment:
The arguments in GeneratingArguments should also be contained in the https://huggingface.co/docs/transformers/v4.40.2/en/main_classes/text_generation#transformers.GenerationConfig, but it is not now, we will delete it